university_has_department.py

```diff
@@ -0,0 +1,18 @@
+class Department:
+    name="jntuk"
+    def __init__(self,branch,code):
+        print("department details")
+        self.branch=branch
+        self.code=code
+    def dept(self):
+        print("ece")
+class university:
+    def __init__(self,branch,code):
+        self.department=Department(branch,code)
```

Comment:
take some instance variables for university as well

Reply:
Ok sir